tools/lint/rules.py

```diff
@@ -135,26 +139,42 @@ class NonexistentRef(Rule):
 class MultipleTimeout(Rule):
     name = "MULTIPLE-TIMEOUT"
     description = "More than one meta name='timeout'"
+    to_fix = """
+        ensure each test file has only one instance of a `<meta
+        name="timeout"...>` element
+    """
 
 
 class InvalidTimeout(Rule):
     name = "INVALID-TIMEOUT"
-    description = "Invalid timeout value %s"
+    description = (
```

Comment:
Why aren't we just using a triple-quoted string here, as we did in earlier versions of the PR?

Reply:
Multi-line strings are fine when rendering markup, but they aren't appropriate for the CLI. Since multi-line strings are easier to maintain, I experimented with normalizing the values at runtime using Python's standard library (specifically `textwrap.dedent` and `inspect.cleandoc`). That takes care of indentation, but the interstitial newlines are still a problem.

Rather than further increase complexity with custom formatting code, I opted for the uglier syntax.